docs/pages/management/dynamic-resources/user-and-role.mdx

```diff
@@ -0,0 +1,501 @@
+---
+title: Managing users and roles with IaC
+description: Use Infrastructure-as-Code tooling to create Teleport users and roles.
+---
+
+In this guide, you will see how to create users and grant them roles through
+Infrastructure-as-Code (IaC). Teleports supports three ways to dynamically
+create resources from code:
+
+- The Teleport Kubernetes Operator that allows you to manage Teleport resources
+  from Kubernetes
+- The Teleport Terraform Provider that allows you to manage Teleport Resources
+  via Terraform
+- The `tctl` CLI that allows you to manage Teleport resources from your local
+  computer or your CI environment
+
+### Prerequisites
```

Comment:
The first-level heading should be H2. We should bump all headings in the page down a level.